.github/actions/frontend/runtime/e2e_pending_status/action.yml

```diff
@@ -0,0 +1,24 @@
+name: Set the status of the PR when E2E tests have been triggered
+
+inputs:
+  github_token:
+    type: string
+    required: true
+
+runs:
+  using: "composite"
+  steps:
+    - name: e2e_pr_status_inputs
+      shell: bash
+      run: |
+        echo "github_token=${{ inputs.github_token }}"
```

Comment:
Why are we echo'ing this, shouldn't it be a secret?

Reply:
Two things in response to this:

1) The action won't actually echo the value as shown here:

<img width="301" alt="Screenshot 2023-10-27 at 4 18 07 PM" src="https://github.com/JupiterOne/.github/assets/1964654/912b320e-67a5-4c3b-abe9-82aa5cb116fd">

2) I echo out the various inputs in a composite action ([example](https://github.com/JupiterOne/.github/blob/967763531e903131c1dfabc2eca5b37136a0e875/.github/actions/frontend/runtime/magic_url/action.yml#L20C5-L20C5) so I can then target them in my E2E tests ([example](https://github.com/JupiterOne/.github/blob/967763531e903131c1dfabc2eca5b37136a0e875/.github/workflows/test/frontend_runtime_application_manual_e2e_run.test.ts#L64C4-L64C4). In this way, I can ensure that with really complicated workflows, the inputs are all getting mapped to the appropriate composite actions and will also know when there breaking changes made (i.e. an input accidentally gets renamed).